src/ui/store/reducers/editHistory.ts

```diff
@@ -23,7 +22,7 @@ export type EditHistoryUnit = {
     diffs: Diff[];
     pointIndex: number;
     pointState?: unknown;
-    setState: ({state}: {state: unknown}) => AnyAction;
+    setState: ({state}: {state: unknown}) => void;
```

Comment:
Now setState handler here does not return action, because all actions are dispatched inside